styles/index.css

```diff
@@ -1,6 +1,10 @@
-@import "./base.css";
 @import "./colors.css";
-@import "./utility.css";
+@import "./base.css";
 
 @import "./quiz-app-card.css";
+
+@import "./components/app-header.css";
 @import "./components/app-grid.css";
+@import "./components/app-nav.css";
+
+@import "./utility.css";
```

Comment:
I think here we need to start with the following as it is working from top to bottom:+
1. colors
2. base
3. utilities (this one is right now on the very bottom)
4. all components below
or is it not working in your case? then we might ask agin 🤔

Reply:
Many thanks that you paid attention on this - in the code it is like you write at the beginning - in the pull request there were only displayed the changes in these lines